autogpt/llm/base.py

```diff
@@ -44,15 +44,11 @@ class ModelInfo:
 class ChatModelInfo(ModelInfo):
     """Struct for chat model information."""
 
-    pass
```

Comment:
what setting is removing all of these?

Reply:
yeah you can't avoid that. they have an ignore pass option, but it's making it even worse: it's adding `pass` everywhere the import was removed.